### ISSUE TITLE
[Accessibility Audit] Integrations: Interacting with an extension on the file page (action item)

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34056]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1_IGekkIDuQugFA-2BYYWpYWpJy9O8PjiDrEEGmJxRRU/edit?usp=sharing). Use this for further context.

- Navigate to this file: https://sourcegraph.com/github.com/sourcegraph/go-diff@9d1f353a285b3094bc33bdae277a19aedabe8b71/-/blob/cmd/go-diff/go-diff.go
- Observe the action items panel on the right side of the code view.
- Click the "Git blame" action item.
- Expect the "Git blame" action item to trigger an action.
  - Note: We cannot guarantee WCAG 2.1 compliance for the actions of these extensions, as they can be created by third-parties.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
